src/Symfony/Component/Translation/Translator.php

```diff
@@ -376,25 +372,37 @@ private function getFallbackContent(MessageCatalogue $catalogue)
             $fallbackSuffix = ucfirst(preg_replace($replacementPattern, '_', $fallback));
             $currentSuffix = ucfirst(preg_replace($replacementPattern, '_', $current));
 
-            $fallbackContent .= sprintf(<<<EOF
-\$catalogue%s = new MessageCatalogue('%s', %s);
-\$catalogue%s->addFallbackCatalogue(\$catalogue%s);
-
-EOF
-                ,
-                $fallbackSuffix,
-                $fallback,
-                var_export($fallbackCatalogue->all(), true),
-                $currentSuffix,
-                $fallbackSuffix
-            );
+            $fallbackContent = $this->getCataloguePhp($fallback, $fallbackCatalogue->all(), $fallbackSuffix);
```

Comment:
Overwriting the ``$fallbackContent`` here breaks things, as it replace the content set in the previous iteration. Please use a different variable name (it is not the content itself either btw)